api/webhook.go

```diff
@@ -21,10 +21,12 @@ func InitWebhook() {
 	l4g.Debug(utils.T("api.webhook.init.debug"))
 
 	BaseRoutes.Hooks.Handle("/incoming/create", ApiUserRequired(createIncomingHook)).Methods("POST")
+	BaseRoutes.Hooks.Handle("/incoming/update", ApiUserRequired(updateIncomingHook)).Methods("PUT")
```

Comment:
Thanks @mpoornima for the update!  And no rush on this, just wanted to make sure you weren't blocked on anything.

Hope you have a good vacation!